src/convert/metadataConverter.ts

```diff
@@ -119,7 +119,7 @@ export class MetadataConverter {
 
       const conversionPipeline = pipeline(
         Readable.from(components),
-        !targetFormatIsSource && (process.env.SF_APPLY_REPLACEMENTS_ON_CONVERT === 'true' || output.type === 'zip')
+        !targetFormatIsSource && output.type === 'zip' && process.env.SF_APPLY_REPLACEMENTS_ON_CONVERT !== 'false'
```

Comment:
SDR was still trying to do the string replacements even with the env var set to false:
```
➜  dreamhouse-lwc git:(main) ✗ SF_APPLY_REPLACEMENTS_ON_CONVERT=false sf deploy
metadata -m ApexClass:GeocodingService
Warning: This command is currently in beta. Any aspect of this command can chang
e without advanced notice. Don't use beta commands in your scripts.
*** Deploying v55.0 metadata with SOAP API v57.0 connection ***
Deploy ID: *************
Status: Failed | ████████████████████████████████████████ | 0/0 Components

Component Failures [4]
=====================================================================
| Type  Name             Problem
| ───── ──────────────── ────────────────────────────────────────────
| Error GeocodingService Expecting '}' but was: ';' (2:60)
| Error GeocodingService Variable does not exist: geocodingUrl (2:46)
| Error GeocodingService Missing ';' at '}' (2:58)
| Error GeocodingService Unexpected token '{'. (2:44
```